packages/ember-cookies/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-cookies",
-  "version": "1.1.2",
```

Comment:
Version shouldn't be changed. It's automatically incremented during release.

Reply:
I updated the version to remind you that this is a new feature and as such merits a minor version update. Removing the version bump.